src/paperbot/api/routes/harvest.py

```
@@ -0,0 +1,448 @@
+    finally:
+        await pipeline.close()
+
+
+@router.post("/harvest")
+async def harvest_papers(request: HarvestRequest):
+    """
+    Harvest papers from multiple sources.
+
+    Returns Server-Sent Events with progress updates.
+    """
+    trace_id = set_trace_id()
```

Ruff (0.15.0)

[error] Local variable `trace_id` is assigned to but never used

Remove assignment to unused variable `trace_id`

(F841)